### ISSUE TITLE
Bootstrap theme switcher?

### ISSUE THREAD

**Issue by cb88** [kristopolous/BOOTSTRA.386#57]
So I used a slightly modified version of https://github.com/jguadagno/bootstrapThemeSwitcher

Would it be possible to get BOOTSTRA.386 working with this? I suspect it would require me to jump through a few hoops since most of my themes don't require swapping out the JS only the CSS file.

**Comment by kristopolous** [comment]
I've read this but I'm at work right now. I'll try to remember to get you an answer this weekend.

**Comment by kristopolous** [comment]
maybe getting this on bootswatch would be the best approach? let me email them

**Comment by kristopolous** [comment]
so I emailed the guy and this theme may be integrated on the site eventually.  However, looking at the project you linked to (https://github.com/jguadagno/bootstrapThemeSwitcher/blob/master/jquery.bootstrapThemeSwitcher.js) this looks like a pretty easy thing to point to wherever you want ... did you try to make this code actually do what you were trying to achieve?

**Comment by cb88** [comment]
Yes I did try getting it working, I probably missed something however as it didn't work.
Most of the themes don't use a separate or modified js file as far as I know... do they?


    On Monday, June 6, 2016 4:14 AM, ᴄʜʀɪꜱ ᴍᴄᴋᴇɴᴢɪᴇ <notifications@github.com> wrote:
 

 so I emailed the guy and this theme may be integrated on the site eventually. However, looking at the project you linked to (https://github.com/jguadagno/bootstrapThemeSwitcher/blob/master/jquery.bootstrapThemeSwitcher.js) this looks like a pretty easy thing to point to wherever you want ... did you try to make this code actually do what you were trying to achieve?—

**Comment by kristopolous** [comment]
no the js isn't required.  Also, try to reply on github and not in email ... it puts the whole thread in the github comment.  If you can post it somewhere I can take a peak.
